ci-operator/config/openshift-pipelines/pipelines-service/openshift-pipelines-pipelines-service-main.yaml

```
@@ -0,0 +1,35 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: plnsvc-test
+  steps:
+    cluster_profile: aws
+    test:
+    - as: plnsvc-test
+      cli: latest
+      commands: ls;echo "onboarding pipelines-service repo"
```

it's a good practice to create a Makefile in the repo with some target you'd run from here, i.e. 
```suggestion
      commands: make test"
```
that would take care of the full test setup and execution